.github/dependabot.yml

```diff
@@ -16,14 +16,21 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ## group all dependencies with a k8s.io prefix into a single PR.
+    groups:
+      kubernetes:
+        patterns: [ "k8s.io/*" ]
     ignore:
       # Ignore controller-runtime as its upgraded manually.
       - dependency-name: "sigs.k8s.io/controller-runtime"
+        update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
       # Ignore k8s and its transitives modules as they are upgraded manually
       # together with controller-runtime.
       - dependency-name: "k8s.io/*"
+        update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
       # ignore ipam, as it needs more than just gomod
       - dependency-name: "github.com/metal3-io/ip-address-manager/api"
+        update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
```

Comment:
IPAM patch release bump includes more than go.mod so it is useless to disable ignore for it. We also bump it sooner than dependabot triggers.

Reply:
done